include/megaapi.h

```diff
@@ -13976,7 +14093,25 @@ class MegaApi
          */
         void checkSMSVerificationCode(const char* verificationCode, MegaRequestListener *listener = NULL);
 
+        /**
+         * @brief Requests the contacts that are registered at MEGA (currently verified through SMS)
+         *
+         * The request will return any of the provided contacts that are registered at MEGA, i.e.,
+         * are verified through SMS (currently). Check the unit tests for an example response.
+         *
+         * @param contacts The map of contacts to get registered contacts from
+         * @param listener MegaRequestListener to track this request
+         */
+        void getRegisteredContacts(const MegaStringMap* contacts, MegaRequestListener *listener = NULL);
 
+        /**
+         * @brief Requests the currently available country calling codes
+         *
+         * Check the unit tests for an example response.
```

Comment:
I think we should at least name the request type and which getters on the request are valid on success?   Most consumers of the API would expect to find all the info about using it here.

Reply:
Will add more info